src/Unleash/Variants/VariantUtils.cs

```diff
@@ -8,9 +8,9 @@ namespace Unleash.Variants
 {
     internal class VariantUtils
     {
-        public static Variant SelectVariant(FeatureToggle featureToggle, UnleashContext context, Variant defaultVariant)
+        public static Variant SelectVariant(FeatureToggle featureToggle, UnleashContext context, Variant defaultVariant, List<VariantDefinition> strategyVariants = null)
```

Comment:
In other SDKs, we pass in list of variants and `groupId`, instead of feature toggle now. Just FYI, maybe makes sense also here.